Marlin/Marlin_main.cpp

```diff
@@ -5414,6 +5388,188 @@ inline void gcode_M226() {
   }
 #endif
 
+/*
+* M242 - General I2C Message Interface
+*   A - 4 - 127 7-bit decimal device address
+*   P - 0 - 255 Process ID (See I2C Commands in Configuration_adv.h)
```

Comment:
Should say "See I2C Commands in Voxel8_I2C_Commands.h", right?

Although the commands do still exist in Configuration_adv.h, they should be removed.